service/local_lnd.js

```diff
@@ -1,19 +1,24 @@
 const {existsSync} = require('fs');
 const {join} = require('path');
 const {readFileSync} = require('fs');
+const dns = require('dns-sync');
 
 const {lightningDaemon} = require('./../lightning');
 
 const adminMacaroonFileName = 'admin.macaroon';
 const chainDirName = 'chain';
 const dataDirName = 'data';
 const lndGrpcPort = process.env.LND_GRPC_PORT || 10009;
-const lndHost = process.env.LND_HOST || 'localhost';
 const {LNSERVICE_CHAIN} = process.env;
 const {LNSERVICE_LND_DIR} = process.env;
 const {LNSERVICE_NETWORK} = process.env;
 const tlsCertFileName = 'tls.cert';
 
+let lndHost = process.env.LND_HOST || 'localhost';
+if (!isIPAddress(lndHost)) {
```

Comment:
where is this function coming from?

Reply:
Sorry was throwing this together to get a PR up for review.  Updated with a regex to test for ip addresses